src/main/java/org/padaiyal/aayvalar/parameterconverters/StringArrayConverter.java

```diff
@@ -0,0 +1,95 @@
+package org.padaiyal.aayvalar.parameterconverters;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Objects;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.params.converter.ArgumentConversionException;
+import org.junit.jupiter.params.converter.SimpleArgumentConverter;
+import org.padaiyal.utilities.I18nUtility;
+import org.padaiyal.utilities.PropertyUtility;
+
+/**
+ * Converts a string into arrays of Strings.
+ */
+public class StringArrayConverter extends SimpleArgumentConverter {
```

Comment:
We should include this with padaiyal/jUnitTestExtras.

Reply:
sure. I'll remove this once this class is moved to padaiyal/jUnitTestExtras